tools/jvm-options-parser/src/main/java/org/logstash/launchers/JvmOptionsParser.java

```diff
@@ -176,7 +176,13 @@ private void handleJvmOptions(Optional<Path> jvmOptionsFile, String lsJavaOpts)
             if (isDebugEnabled()) {
                 System.err.println("Appending jvm options from environment LS_JAVA_OPTS");
             }
-            jvmOptionsContent.add(lsJavaOpts);
+            if (lsJavaOpts.contains(" ")) {
+                Arrays.stream(lsJavaOpts.split(" "))
+                        .filter(s -> !s.isBlank())
+                        .forEach(jvmOptionsContent::add);
+            } else {
+              jvmOptionsContent.add(lsJavaOpts);
+            }
```

Comment:
```suggestion
            Arrays.stream(lsJavaOpts.split(" "))
                    .filter(s -> !s.isBlank())
                    .forEach(jvmOptionsContent::add);
```

Can we remove `if` condition?